Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -0,0 +1,1946 @@
+        return [file_]
+    else:
+        return [CommandResults(
+            readable_output=readable_outputs,
+            outputs_prefix='NSM.Attacks',
+            outputs=attacks_list,
+            raw_response=attacks_list,
+            outputs_key_field='ID'
+        )]
+
+
+def get_domains_command(client: Client, args: Dict) -> CommandResults:
+    """ If a domain id is given The command returns the details for the specific domain.
+        Else, gets all available domains.
+        Args:
+            client: client - A McAfeeNSM client.
+            args: Dict - The function arguments.
+        Returns:
+            A CommandResult object with The domain details or domains list.
+    """
+    domain_id = arg_to_number(args.get('domain_id', None))
+    limit = arg_to_number(args.get('limit', 50)) or 50
+    page = arg_to_number(args.get('page'))
+    page_size = arg_to_number(args.get('page_size'))
+    if (page and not page_size) or (not page and page_size):
+        raise Exception('Please provide both page and page_size arguments.')
+    if args.get('limit') and page:
+        raise Exception('Please provide either limit argument or page and page_size arguments.')
+
+    response = client.get_domains_request(domain_id)
+    results = response.get('DomainDescriptor', {})
+    human_readable = []
+    if domain_id is not None:
+        title = f'Domain no.{domain_id}'
+        results = {
+            'ID': results.get('id'),
+            'Name': results.get('name'),
+            'childdomains': results.get('childdomains')
+        }
+        human_readable = [{
+            'ID': results.get('ID'),
+            'Name': results.get('Name')
+        }]
+    else:
+        title = 'List of Domains'
+        children = [results]
+        h_r_get_domains(children, human_readable)
+    human_readable = pagination(human_readable, limit, page, page_size)
+    readable_outputs = tableToMarkdown(
+        name=title,
+        t=human_readable,
+        removeNull=True
+    )
+    return CommandResults(
+        readable_output=readable_outputs,
+        outputs_prefix='NSM.Domains',
+        outputs=results,
+        raw_response=results,
+        outputs_key_field='ID'
+    )
+
+
+def get_sensors_command(client: Client, args: Dict) -> CommandResults:
+    """ Gets the list of sensors available in the specified domain. If the domain is not specified, details of all
+        the sensors in all domains will be provided.
+        Args:
+            client: client - A McAfeeNSM client.
+            args: Dict - The function arguments.
+        Returns:
+            A CommandResult object with The relevant sensors.
+    """
+    domain_id = arg_to_number(args.get('domain_id'))
+    limit = arg_to_number(args.get('limit', 50)) or 50
+    page = arg_to_number(args.get('page'))
+    page_size = arg_to_number(args.get('page_size'))
+    if (page and not page_size) or (not page and page_size):
```

Comment:
We're doing this check several times in the code. Lets create a helper function for it